### DISCUSSION TITLE
How do i add custom icon or social icon as brand icons are depreciated

### DISCUSSION THREAD

**Discussion by khanakia** [lucide-icons/lucide#2245] [Q&A]
As all the brand icons are appreciated

So how do I create a custom icon component from SVG or [simpleicons](https://simpleicons.org/) as everybody is recommending

Thanks

**Comment by ericfennis** [comment]
@khanakia Which framework library are you using?

**Reply by khanakia** [reply]
I am using Next.js and lucide-react library

**Comment by heryckmp** [comment]
is a good library
